src/mode/modeHandler.ts

```diff
@@ -1472,6 +1483,47 @@ export class ModeHandler implements vscode.Disposable {
         }
       }
 
+      // Combine instersected selections - When we have multiple cursors
+      // sometimes those cursors selections intersect and combine, we need
+      // to check that here so that we know if our currents cursors will
+      // trigger a selectionChangeEvent or not. If we didn't check for this
+      // vscode might already have the resulting combined selection selected
+      // but since that wouldn't be the same as our selections we would think
+      // there would be a selectionChangeEvent when there wouldn't be any.
+      let combinedSelections: vscode.Selection[] = [];
```

Comment:
Split this into a separate function (inline if you like) to localize `combinedSelections`

Reply:
What do you mean here exactly with localize `combinedSelections`?